### PR TITLE
User onboarding

### PR DESCRIPTION
**Pull request by TejasShekar**
User Onboarding page

### CODE REVIEW

File: src/components/Time.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  useEffect(() => {
+    setTimeout(() => {
+      const date = new Date();
+      const time = date.toLocaleTimeString("en-GB");
+      setTime(time);
+    }, 1000);
+  });
+  return (
+    <>
+      <h1 className="time">{time.slice(0, 5)}</h1>
```
Comment (sashaboi) [inline src/components/Time.jsx:14]: you could put a loader here in future version till the timeout works, just a suggestion.

**Review by sashaboi — APPROVED** [review]
Looks interesting, looking forward to future versions of the project!

**Comment by poojakelkar** [comment]
code looks clean with proper naming with good UI

**Review by ravikumar0403 — APPROVED** [review]
Clean and very well written code.

**Pull request merged**